languages/go/exercises/concept/using-map/.docs/instructions.md

```diff
@@ -0,0 +1,22 @@
+In this excersie, you'll help uncle Donald to count monthly and total income of his thrift shop, he recorded of all of the transaction in his journal book, but he forgot to put the price it it, here's the price list
```

Comment:
I've tried to rephrase a bit to make it a little bit more clear IMHO:

```suggestion
In this excercise, you'll help uncle Donald calculate his thrift shop's monthly and total income. Your uncle records all transactions in his journal book, but unfortunately none of the entries have a price specified. However, there is a price list for each item that can be used to get a transactions price:
```

Feel free to disagree of course :)